components/brave_wallet/browser/solana_tx_manager.cc

```diff
@@ -927,119 +913,129 @@ void SolanaTxManager::OnGetAccountInfo(
                           mojom::SolanaProviderError::kSuccess, "");
 }
 
-void SolanaTxManager::GetEstimatedTxBaseFee(
```

Comment:
I think `GetEstimatedTxBaseFee` and `GetEstimatedTxBaseFeeCallback` are still leftover in .h